src/mono/msbuild/android/build/AndroidBuild.targets

```diff
@@ -3,9 +3,12 @@
              AssemblyFile="$(AndroidAppBuilderTasksAssemblyPath)" />
 
   <PropertyGroup>
+    <AndroidGenerateAppBundle Condition="'$(AndroidGenerateAppBundle)' == '' and '$(GenerateAppBundle)' != ''">$(GenerateAppBundle)</AndroidGenerateAppBundle>
```

Comment:
Looks like this file will be part of library builder, there is still a bunch of references to App in this file, since its more generic and shared, maybe we should make sure properties are aligned as much as possible to be more generic and not app centric?